.travis.yml

```diff
@@ -3,7 +3,7 @@ language: node_js
 node_js:
   # we recommend testing addons with the same minimum supported node version as Ember CLI
   # so that your addon works for all apps
-  - '6'
+  - '10'
```

Comment:
We should probably use 8 (minimum supported Node) and do this in a separate PR to drop Node 6 support (will be breaking). Will also need to tweak the `engines.node` entry in `package.json`.

Reply:
to get this unblocked in a non-breaking way we could also run the scenarios for Ember 3.13 and above on `node_js: 10` and keep the others at `node_js: 6` for now until we actually drop support

Reply:
I went with @Turbo87's suggestion and pinned the new scenarios at `8`. I'll follow up with another to drop `Node 6`.